platform/src/test/kotlin/org/stellar/anchor/platform/action/NotifyRefundInitiatedHandlerTest.kt

```diff
@@ -342,4 +342,35 @@ class NotifyRefundInitiatedHandlerTest {
     assertTrue(expectedSep24Txn.updatedAt >= startDate)
     assertTrue(expectedSep24Txn.updatedAt <= endDate)
   }
+
+  @Test
+  fun `test handle more then amount_in`() {
```

Comment:
I suggest using camel case for test method name to follow the naming of Handler tests